package.json

```diff
@@ -13,8 +13,8 @@
     "devInstall": "dev-env-installer install"
   },
   "dependencies": {
-    "bignumber.js": "8.1.1",
-    "date-and-time": "0.6.3",
+    "bignumber.js": "9.0.0",
+    "date-and-time": "0.7.0",
     "escape-html": "1.0.3",
     "json-schema-compatibility": "1.1.0",
     "json-to-ast": "2.0.0-alpha1.3",
```

Comment:
Let's not use non stable versions (i.e. `alpha`). There seems to be a newer `2.1.0` version anyway, let's use that or any other stable version.

Reply:
I'll work on this in a separate branch because release 2.0.0 adds a lot of breaking changes.

Reply:
https://github.com/raml-org/raml-typesystem/issues/125